lib/github.ts

```diff
@@ -70,3 +70,46 @@ async function fetchIssues(args: {
   }
   return GitHubIssueResponseSchema.parse(await resp.json());
 }
+
+interface IContributor {
+  login: string;
+  avatar_url: string;
+  html_url: string;
+}
+
+export async function fetchGitHubContributors(token: string, repos: string[]) {
+  const allContributors: IContributor[] = [];
+  const contributorsMap = new Map<string, IContributor>();
+
+  for (const repo of repos) {
+    const response = await fetch(`https://api.github.com/repos/${repo}/contributors`, {
+      headers: {
+        Authorization: `token ${token}`,
+        Accept: 'application/vnd.github.v3+json',
+      },
+    });
+
+    if (!response.ok) {
+      throw new Error(`GitHub API request failed with status ${response.status}`);
+    }
+
+    const contributors = await response.json();
+
+    for (const contributor of contributors) {
+      const contributorData: IContributor = {
+        login: contributor.login,
+        avatar_url: contributor.avatar_url,
+        html_url: contributor.html_url,
+      };
+
+      if (!contributorsMap.has(contributor.login)) {
+        contributorsMap.set(contributor.login, contributorData);
+      }
```

Comment:
Good call using a map to dedupe here :100:

Reply:
Should I exclude dependabot  (bot) as a contribute or leave it there?

Reply:
No big deal either way -- but if you feel like it, filtering it out probably makes the most sense.